File: api/code-samples/javascript/klaytnDebug/vMTracing/traceBlockByHash.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));
    
    const blockHash = "0x651722eb826af57fd95a2381c9cc0c162f90087d8283d02945c42b48229edf86";

    sdk.debug.traceBlockByHash(blockHash, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: api/code-samples/python/klaytnDebug/vMTracing/traceBlockByHash.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

blockHash = "0xed110b330152df2022d40fa3c38987643034aa56fc96079fb6c67b66a6ed4f19"

sdk = OpenSDK(host)
debug_response = sdk.debug.trace_block_by_hash(blockHash)

print(json.loads(debug_response.response.data))
